todoman/formatters.py

```diff
@@ -85,13 +85,23 @@ def compact_multiple(self, todos):
 
         return tabulate(table, tablefmt='plain')
 
-    def _columnize(self, label, text):
+    def _columnize_text(self, label, text):
+        """Display text, split text by line-endings, on multiple colums,"""
+        """do nothing if text is empty or None"""
+        lines = text.splitlines() if text else None
+
+        return self._columnize_list(label, lines)
+
+    def _columnize_list(self, label, lst):
```

Comment:
I'd move this function's logic inside `_columnize_text`, since it's not really something we're reusing.

Reply:
I externalized this part on purpose, because we will use it for the categories. (see PR #323)